mux.PGO.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <PlatformShortName Condition="'$(PlatformShortName)'==''">$(Platform)</PlatformShortName>
```

Comment:
What is the purpose of 'PlatformShortName' over 'Platform'?

Reply:
Win32 vs x86

---
In reply to: [301742511](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301742511) [](ancestors = 301742511)